hazelcast/src/main/java/com/hazelcast/internal/server/tcp/TcpServerConnectionManager.java

```diff
@@ -76,87 +58,40 @@
 import static com.hazelcast.internal.nio.ConnectionType.REST_CLIENT;
 import static com.hazelcast.internal.nio.IOUtil.close;
 import static com.hazelcast.internal.nio.IOUtil.setChannelOptions;
-import static com.hazelcast.internal.util.ConcurrencyUtil.getOrPutIfAbsent;
 import static com.hazelcast.internal.util.Preconditions.checkNotNull;
-import static com.hazelcast.internal.util.counters.MwCounter.newMwCounter;
-import static com.hazelcast.spi.properties.ClusterProperty.CHANNEL_COUNT;
 import static java.lang.Math.abs;
 import static java.util.Arrays.stream;
-import static java.util.Collections.newSetFromMap;
 import static java.util.Collections.unmodifiableSet;
 
-@SuppressWarnings("checkstyle:methodcount")
-public class TcpServerConnectionManager
+/**
+ * This is the public APIs for connection manager
+ * Private APIs and supporting stuff is in the Base class
+ */
+public class TcpServerConnectionManager extends TcpServerConnectionManagerBase
         implements ServerConnectionManager, Consumer<Packet>, DynamicMetricsProvider {
```

Comment:
`implements ServerConnectionManager` is redundant here (the base class also implements this interface)

Reply:
fixed